src/workerd/io/worker.h

```diff
@@ -711,8 +712,18 @@ class Worker::Actor final: public kj::Refcounted {
   // Runs an alarm in this actor. This function will handle deduplicating multiple alarms. If this
   // isn't a duplicate alarm, func will be called to run the alarm.
 
+  kj::Own<Worker::Actor> addRef() {
+    KJ_IF_MAYBE(t, tracker) {
+      return kj::addRef(*this).attach(t->startRequest());
+    } else {
+      return kj::addRef(*this);
+    }
+  }
 private:
   kj::Own<const Worker> worker;
+
+  // TODO(soon): Provide a RequestTracker in workerd so this can no longer be a Maybe
```

Comment:
Hmmmm, what do you have in mind? It would be nice if we could just take care of it in this PR.